src/include/miopen/solver.hpp

```diff
@@ -982,13 +982,27 @@ struct ConvHipImplicitGemmMlirCppWrW : SolverBase<ConvolutionContext>
 struct ConvMlirIgemmWrW : SolverBase<ConvolutionContext>
 {
     bool IsApplicable(const ConvolutionContext& ctx) const;
-    ConvSolution GetSolution(const ConvolutionContext& ctx) const;
+    PerformanceConvMlirIgemm GetPerformanceConfig(const ConvolutionContext& ctx) const;
+    bool IsValidPerformanceConfig(const ConvolutionContext& ctx,
+                                  const PerformanceConvMlirIgemm& config) const;
+    PerformanceConvMlirIgemm Search(const ConvolutionContext&,
+                                    const AnyInvokeParams& invoke_ctx) const;
+    ConvSolution GetSolution(const ConvolutionContext& ctx,
+                             const PerformanceConvMlirIgemm& config,
+                             bool disableConfigOverrideFromEnv = false) const;
```

Comment:
[Quality] In all tunable MLIR solvers: If you do not use overriding of PerformanceConfigs from the environment (this is a debug feature implemented in some solvers for testing), then maybe
```suggestion
                             bool) const;
```

Reply:
I see. Yes, C++ wants the default in this case.